Sanity tests: resolve pylint errors

```ERROR: Found 20 validate-modules issue(s) which need to be resolved:
ERROR: lib/ansible/modules/cloud/memset/memset_dns_reload.py:0:0: E105 GPLv3 license header not found in the first 20 lines of the module
ERROR: lib/ansible/modules/cloud/memset/memset_dns_reload.py:0:0: E316 ANSIBLE_METADATA.metadata_version: not a valid value for dictionary value @ data['metadata_version']. Got '1.0'
ERROR: lib/ansible/modules/cloud/memset/memset_dns_reload.py:4:0: E106 Import found before documentation variables. All imports must appear below DOCUMENTATION/EXAMPLES/RETURN/ANSIBLE_METADATA.
ERROR: lib/ansible/modules/cloud/memset/memset_dns_reload.py:24:79: E302 DOCUMENTATION is not valid YAML
ERROR: lib/ansible/modules/cloud/memset/memset_zone.py:0:0: E105 GPLv3 license header not found in the first 20 lines of the module
ERROR: lib/ansible/modules/cloud/memset/memset_zone.py:0:0: E316 ANSIBLE_METADATA.metadata_version: not a valid value for dictionary value @ data['metadata_version']. Got '1.0'
ERROR: lib/ansible/modules/cloud/memset/memset_zone.py:4:0: E106 Import found before documentation variables. All imports must appear below DOCUMENTATION/EXAMPLES/RETURN/ANSIBLE_METADATA.
ERROR: lib/ansible/modules/cloud/memset/memset_zone.py:24:77: E302 DOCUMENTATION is not valid YAML
ERROR: lib/ansible/modules/cloud/memset/memset_zone.py:53:9: E311 EXAMPLES is not valid YAML
ERROR: lib/ansible/modules/cloud/memset/memset_zone_domain.py:0:0: E105 GPLv3 license header not found in the first 20 lines of the module
ERROR: lib/ansible/modules/cloud/memset/memset_zone_domain.py:0:0: E316 ANSIBLE_METADATA.metadata_version: not a valid value for dictionary value @ data['metadata_version']. Got '1.0'
ERROR: lib/ansible/modules/cloud/memset/memset_zone_domain.py:4:0: E106 Import found before documentation variables. All imports must appear below DOCUMENTATION/EXAMPLES/RETURN/ANSIBLE_METADATA.
ERROR: lib/ansible/modules/cloud/memset/memset_zone_domain.py:24:77: E302 DOCUMENTATION is not valid YAML
ERROR: lib/ansible/modules/cloud/memset/memset_zone_domain.py:48:11: E311 EXAMPLES is not valid YAML
ERROR: lib/ansible/modules/cloud/memset/memset_zone_record.py:0:0: E105 GPLv3 license header not found in the first 20 lines of the module
ERROR: lib/ansible/modules/cloud/memset/memset_zone_record.py:0:0: E316 ANSIBLE_METADATA.metadata_version: not a valid value for dictionary value @ data['metadata_version']. Got '1.0'
ERROR: lib/ansible/modules/cloud/memset/memset_zone_record.py:4:0: E106 Import found before documentation variables. All imports must appear below DOCUMENTATION/EXAMPLES/RETURN/ANSIBLE_METADATA.
ERROR: lib/ansible/modules/cloud/memset/memset_zone_record.py:24:77: E302 DOCUMENTATION is not valid YAML
ERROR: lib/ansible/modules/cloud/memset/memset_zone_record.py:71:12: E311 EXAMPLES is not valid YAML
ERROR: lib/ansible/modules/cloud/memset:0:0: E502 Ansible module subdirectories must contain an __init__.py
```

--- a/lib/ansible/modules/cloud/memset/memset_dns_reload.py
+++ b/lib/ansible/modules/cloud/memset/memset_dns_reload.py
@@ -1,12 +1,14 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, Simon Weald <ansible@simonweald.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
-from time import sleep
-from ansible.module_utils.memset import memset_api_call
 __metaclass__ = type
 
 ANSIBLE_METADATA = {
-    'metadata_version': '1.0',
+    'metadata_version': '1.1',
     'status': ['preview'],
     'supported_by': 'community'
 }
@@ -15,7 +17,7 @@ DOCUMENTATION = '''
 ---
 module: memset_dns_reload
 author: "Simon Weald (@analbeard)"
-version_added: "2.5"
+version_added: "2.6"
 short_description: Request reload of Memset's DNS infrastructure
 notes:
   - DNS reload requests are a best-effort service provided by Memset; these generally
@@ -80,6 +82,9 @@ memset_api:
       type: string
       sample: "dns"
 '''
+
+from time import sleep
+from ansible.module_utils.memset import memset_api_call
 
 
 def reload_dns(args=None):

--- a/lib/ansible/modules/cloud/memset/memset_dns_reload.py
+++ b/lib/ansible/modules/cloud/memset/memset_dns_reload.py
@@ -82,7 +82,7 @@ memset_api:
 '''
 
 
-def reload_dns(args):
+def reload_dns(args=None):
     retvals, payload = dict(), dict()
     has_changed, has_failed = False, False
     memset_api, msg = None, None
@@ -126,7 +126,7 @@ def reload_dns(args):
     return(retvals)
 
 
-def main(args=dict()):
+def main():
     global module
     module = AnsibleModule(
         argument_spec=dict(
@@ -136,6 +136,7 @@ def main(args=dict()):
         supports_check_mode=False
     )
 
+    args = dict()
     args['api_key'] = module.params['api_key']
     args['poll'] = module.params['poll']
 

--- a/lib/ansible/modules/cloud/memset/memset_zone.py
+++ b/lib/ansible/modules/cloud/memset/memset_zone.py
@@ -1,13 +1,14 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, Simon Weald <ansible@simonweald.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
-from ansible.module_utils.memset import check_zone
-from ansible.module_utils.memset import get_zone_id
-from ansible.module_utils.memset import memset_api_call
 __metaclass__ = type
 
 ANSIBLE_METADATA = {
-    'metadata_version': '1.0',
+    'metadata_version': '1.1',
     'status': ['preview'],
     'supported_by': 'community'
 }
@@ -16,7 +17,7 @@ DOCUMENTATION = '''
 ---
 module: memset_zone
 author: "Simon Weald (@analbeard)"
-version_added: "2.5"
+version_added: "2.6"
 short_description: Manage zones
 notes:
   - Zones can be thought of as a logical group of domains, all of which share the
@@ -98,6 +99,10 @@ memset_api:
       type: int
       sample: 300
 '''
+
+from ansible.module_utils.memset import check_zone
+from ansible.module_utils.memset import get_zone_id
+from ansible.module_utils.memset import memset_api_call
 
 
 def check(args=None):

--- a/lib/ansible/modules/cloud/memset/memset_zone.py
+++ b/lib/ansible/modules/cloud/memset/memset_zone.py
@@ -100,7 +100,7 @@ memset_api:
 '''
 
 
-def check(args):
+def check(args=None):
     # get the zones and check if the relevant zone exists
     retvals = dict()
 
@@ -115,12 +115,12 @@ def check(args):
     retvals['changed'] = has_changed
     retvals['failed'] = has_failed
     retvals['memset_api'] = response.json()
-    retvals['msg'] = 'Zone "{}" exists: {}' . format(args['name'], str(zone_exists))
+    retvals['msg'] = 'Zone "{0}" exists: {1}' . format(args['name'], str(zone_exists))
 
     return(retvals)
 
 
-def create_or_delete(args):
+def create_or_delete(args=None):
     retvals = dict()
     has_failed, has_changed = False, False
     msg, memset_api, _stderr = None, None, None
@@ -198,7 +198,7 @@ def create_or_delete(args):
     return(retvals)
 
 
-def main(args=dict()):
+def main():
     global module
     module = AnsibleModule(
         argument_spec=dict(
@@ -211,6 +211,7 @@ def main(args=dict()):
         supports_check_mode=True
     )
 
+    args = dict()
     args['state'] = module.params['state']
     args['api_key'] = module.params['api_key']
     args['name'] = module.params['name']

--- a/lib/ansible/modules/cloud/memset/memset_zone_domain.py
+++ b/lib/ansible/modules/cloud/memset/memset_zone_domain.py
@@ -1,13 +1,14 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, Simon Weald <ansible@simonweald.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
-from ansible.module_utils.memset import get_zone_id
-from ansible.module_utils.memset import check_zone_domain
-from ansible.module_utils.memset import memset_api_call
 __metaclass__ = type
 
 ANSIBLE_METADATA = {
-    'metadata_version': '1.0',
+    'metadata_version': '1.1',
     'status': ['preview'],
     'supported_by': 'community'
 }
@@ -16,7 +17,7 @@ DOCUMENTATION = '''
 ---
 module: memset_zone_domain
 author: "Simon Weald (@analbeard)"
-version_added: "2.5"
+version_added: "2.6"
 short_description: Manage zone domains
 notes:
   - Zone domains can be thought of as a collection of domains, all of which share the
@@ -69,6 +70,10 @@ memset_api:
       type: string
       sample: "b0bb1ce851aeea6feeb2dc32fe83bf9c"
 '''
+
+from ansible.module_utils.memset import get_zone_id
+from ansible.module_utils.memset import check_zone_domain
+from ansible.module_utils.memset import memset_api_call
 
 
 def check(args=None):

--- a/lib/ansible/modules/cloud/memset/memset_zone_domain.py
+++ b/lib/ansible/modules/cloud/memset/memset_zone_domain.py
@@ -71,7 +71,8 @@ memset_api:
 '''
 
 
-def check(args, retvals=dict()):
+def check(args=None):
+    retvals = dict()
     api_method = 'dns.zone_domain_list'
     has_changed = False
 
@@ -88,7 +89,8 @@ def check(args, retvals=dict()):
     return(retvals)
 
 
-def create_or_delete_domain(args, retvals=dict()):
+def create_or_delete_domain(args=None):
+    retvals = dict()
     has_changed, has_failed = False, False
     msg, stderr, memset_api = None, None, None
     payload = dict()
@@ -100,16 +102,16 @@ def create_or_delete_domain(args, retvals=dict()):
     if has_failed:
         retvals['failed'] = has_failed
         retvals['msg'] = msg
-        retvals['stderr'] = "API returned an error: {}" .format(response.status_code)
+        retvals['stderr'] = "API returned an error: {0}" .format(response.status_code)
         return(retvals)
 
     zone_exists, msg, counter, zone_id = get_zone_id(zone_name=args['zone'], current_zones=response.json())
     if not zone_exists:
         has_failed = True
         if counter == 0:
-            stderr = "DNS zone '{}' does not exist, cannot create domain." . format(args['zone'])
+            stderr = "DNS zone '{0}' does not exist, cannot create domain." . format(args['zone'])
         elif counter > 1:
-            stderr = "{} matches multiple zones, cannot create domain." . format(args['zone'])
+            stderr = "{0} matches multiple zones, cannot create domain." . format(args['zone'])
 
         retvals['failed'] = has_failed
         retvals['msg'] = stderr
@@ -155,7 +157,7 @@ def create_or_delete_domain(args, retvals=dict()):
     return(retvals)
 
 
-def main(args=dict()):
+def main():
     global module
     module = AnsibleModule(
         argument_spec=dict(
@@ -167,6 +169,7 @@ def main(args=dict()):
         supports_check_mode=True
     )
 
+    args = dict()
     args['state'] = module.params['state']
     args['api_key'] = module.params['api_key']
     args['domain'] = module.params['domain']

--- a/lib/ansible/modules/cloud/memset/memset_zone_record.py
+++ b/lib/ansible/modules/cloud/memset/memset_zone_record.py
@@ -138,9 +138,10 @@ memset_api:
 '''
 
 
-def create_or_delete(args, payload=dict()):
+def create_or_delete(args=None):
     has_failed, has_changed = False, False
     msg, memset_api, stderr = None, None, None
+    payload = dict()
     retvals = dict()
 
     # get the zones and check if the relevant zone exists
@@ -254,7 +255,7 @@ def create_or_delete(args, payload=dict()):
     return(retvals)
 
 
-def main(args=dict()):
+def main():
     global module
     module = AnsibleModule(
         argument_spec=dict(
@@ -271,6 +272,7 @@ def main(args=dict()):
         supports_check_mode=True
     )
 
+    args = dict()
     args['state'] = module.params['state']
     args['api_key'] = module.params['api_key']
     args['zone'] = module.params['zone']

--- a/lib/ansible/modules/cloud/memset/memset_zone_record.py
+++ b/lib/ansible/modules/cloud/memset/memset_zone_record.py
@@ -1,13 +1,14 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, Simon Weald <ansible@simonweald.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
-from ansible.module_utils.memset import get_zone_id
-from ansible.module_utils.memset import memset_api_call
-from ansible.module_utils.memset import get_zone_id
 __metaclass__ = type
 
 ANSIBLE_METADATA = {
-    'metadata_version': '1.0',
+    'metadata_version': '1.1',
     'status': ['preview'],
     'supported_by': 'community'
 }
@@ -16,7 +17,7 @@ DOCUMENTATION = '''
 ---
 module: memset_zone_record
 author: "Simon Weald (@analbeard)"
-version_added: "2.5"
+version_added: "2.6"
 short_description: Manage zone records
 notes:
   - Zones can be thought of as a logical group of domains, all of which share the
@@ -136,6 +137,10 @@ memset_api:
       type: string
       sample: "b0bb1ce851aeea6feeb2dc32fe83bf9c"
 '''
+
+from ansible.module_utils.memset import get_zone_id
+from ansible.module_utils.memset import memset_api_call
+from ansible.module_utils.memset import get_zone_id
 
 
 def create_or_delete(args=None):

